save-cli/src/nativeMain/kotlin/org/cqfn/save/cli/SaveConfigReader.kt

```diff
@@ -0,0 +1,83 @@
+/**
+ * Utilities to work with SAVE config in CLI mode
+ */
+
+package org.cqfn.save.cli
+
+import org.cqfn.save.cli.logging.logErrorAndExit
+import org.cqfn.save.core.logging.logDebug
+import org.cqfn.save.core.logging.logInfo
+
+import okio.FileSystem
+import okio.IOException
+import okio.Path.Companion.toPath
+
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.properties.Properties
+import kotlinx.serialization.serializer
+import okio.FileNotFoundException
+import org.cqfn.save.core.Save
+import org.cqfn.save.core.config.*
+
+/**
+ * @param args CLI args
+ * @return an instance of [SaveCliConfig]
```

Comment:
What name do we choose? `SaveConfig` or `SaveCliConfig`?

Reply:
SaveProperties and TestConfig